flink-end-to-end-tests/run-pre-commit-tests.sh

```diff
@@ -51,6 +51,7 @@ run_test "Shaded Hadoop S3A end-to-end test" "$END_TO_END_DIR/test-scripts/test_
 run_test "Shaded Presto S3 end-to-end test" "$END_TO_END_DIR/test-scripts/test_shaded_presto_s3.sh"
 run_test "Hadoop-free Wordcount end-to-end test" "$END_TO_END_DIR/test-scripts/test_hadoop_free.sh"
 run_test "Distributed cache end-to-end test" "$END_TO_END_DIR/test-scripts/test_streaming_distributed_cache_via_blob.sh"
+run_test "Wordcount end-to-end test in docker env" "$END_TO_END_DIR/test-scripts/test_docker_embedded_job.sh"
```

Comment:
if we can't run it on travis it shouldn't be in this script?

Reply:
We can run docker test on travis. We can't run the kubernetes one.

Reply:
The test failed because of known issue that after stopping the job occassionally `org.apache.flink.util.FlinkException: The assigned slot 303c27bfb546addd31de16e1af05509a_0 was removed.` is logged.